nixos/modules/services/matrix/mautrix-discord.nix

```diff
@@ -0,0 +1,130 @@
+{ config, pkgs, lib, ... }:
```

Comment:
Please split this addition into a separate commit (`nixos/mautrix-discord: init` or similar).